### ISSUE TITLE
Cropped windows and JS errors when entering gnome shell overview

### ISSUE THREAD

**Issue by aleeraser** [home-sweet-gnome/dash-to-panel#1580]
**Bug description**
I'm getting the following errors while triggering the gnome shell overview (with the super key):

```JS ERROR: TypeError: area is null
padArea@resource:///org/gnome/shell/ui/workspace.js:1101:9
_updateWindowPositions@resource:///org/gnome/shell/ui/workspace.js:1334:20
_realRecalculateWindowPositions@resource:///org/gnome/shell/ui/workspace.js:1311:14
_recalculateWindowPositions/this._positionWindowsId<@resource:///org/gnome/shell/ui/workspace.js:1286:18
```

**System**
- Ubuntu 20.04.4 lts
- gnome shell 3.36.9
- dash to panel 42, installed from gnome extensions website

<br/>

**Steps to Reproduce**
1. login to session
2. press super

Expected result: no errors and properly resized windows.
Actual result: JS errors and cropped windows that can be seen in the attached screen capture.

<br/>

**Video capture**
[![video capture](https://user-images.githubusercontent.com/10846607/155115836-9e4be86c-693e-498f-95cb-09690120f2f6.png)](https://user-images.githubusercontent.com/10846607/155114897-6d2b6a5e-5c23-4d7f-b5e5-297a3005217a.mp4)

<br/>

**Additional Info**
I verified that the root cause of the problem is this extension by disabling all the other extensions before trying (as shown in the screen capture).

**Comment by charlesg99** [comment]
Could you disable all your extensions like you did but then restart gnome-shell and see if the issue occurs? The error you provided isn't from dash to panel and I've never seen this behavior. Thanks!

**Comment by aleeraser** [comment]
I tried to trigger the various possible combinations of no exts | no exts. but Dash-to-panel | all exts. excep Dash-to-Panel | all extensions, while restarting gnome-shell in between.

I experienced this issue almost every time, and when I hit it Dash-to-Panel was enabled.

But... I did the process one last time, and unfortunately I don't exactly recall the combination of events I went through, but now the issue is gone. I have all the exts. enabled including Dash-to-Panel, I also triggered a logout/login and a reboot, but pressing the super key and entering the gnome-shell overview now correctly re-arranges the windows.

I have no clue why and how the issue was happening, nor how it got solved. Bear in mind that I've had this weird behavior every day since the time I opened this issue, one and a half months ago (which means meanwhile the laptop went through reboots, gnome-shell restarts, and so on).

I guess this can be closed.
